src/Task02_PokeAPI/pokeApi.js

```diff
@@ -0,0 +1,172 @@
+export async function fetchPokemonData (pokemonName) {
+  const graphqlQuery = {
+    query: `
+      query getPokemonDetails($name: String!) {
+        pokemon(name: $name) {
+          name
+          weight
+          height
+          base_experience
+          sprites {
+            front_default
+            back_default
+          }
+          types {
+            type {
+              name
+            }
+          }
+          abilities {
+            ability {
+              name
+            }
+          }
+          stats {
+            stat {
+              name
+            }
+          }
+          location_area_encounters
+        }
+      }
+    `,
+    variables: { name: pokemonName.toLowerCase() }
+  }
+
+  try {
+    const response = await fetch('https://graphql-pokeapi.graphcdn.app/', {
```

Comment:
put api urls to constants. better to keep in some shared file and then import. in that case you will be sure you are using the same url and won't have bugs